src/auth-service.ts

```diff
@@ -296,15 +294,15 @@ export async function signUp({
   email: string
   firstName?: string
   lastName?: string
-}): Promise<FormsAppUser> {
+}): Promise<boolean> {
```

Comment:
Can just return void, the api will return an error code which will throw an exception if the sign up fails